examples/knative/powershell/kn-ps-vrni-databus/handler.ps1

```diff
@@ -0,0 +1,84 @@
+Import-Module CloudEvents.Sdk
+
+. ./vrni-functions.ps1
+
+Function Process-Init {
+   [CmdletBinding()]
+   param()
+   Write-Host "$(Get-Date) - Processing Init`n"
+
+   Write-Host "$(Get-Date) - Init Processing Completed`n"
+}
+
+Function Process-Shutdown {
+   [CmdletBinding()]
+   param()
+   Write-Host "$(Get-Date) - Processing Shutdown`n"
+
+   Write-Host "$(Get-Date) - Shutdown Processing Completed`n"
+}
+
+Function Process-Handler {
+   [CmdletBinding()]
+   param(
+      [Parameter(Position = 0, Mandatory = $true)]$Source,
+      [Parameter(Position = 1, Mandatory = $true)]$Body
+   )
+
+   # Retrieve webhook sink URL
+   $webhookSinkUrl = ${env:WEBHOOK_SINK_URL}
+
+   Write-host "$(Get-Date) - Creating CloudEvents...`n"
+   try {
+      $cloudEvents = New-vRNICloudEventsFromDatabus -Data $body -cloudEventSource $Source
+   }
+   catch {
+      $ErrorMessage = $_.Exception.Message
+      Write-Host "$(Get-Date) - Unable to run New-vRNICloudEventsFromDatabus: "
+      Write-Host $ErrorMessage
+   }
+
+   if ($cloudEvents.Count -eq 0) {
+      Write-Host "$(Get-Date) - No cloud events generated: not sending to VMware Event Broker"
+      return
+   }
+
+   # Set the default broker internal forwarding URL if it's not given
+   if ($null -ne $webhookSinkUrl -and $webhookSinkUrl -ne "") {
+      $webhookSinkUrl = "http://default-broker-ingress.vmware-functions.svc.cluster.local"
+   }
+   # Disable forwarding to broker if $ENV:SERVICE_TEST is true
```

Comment:
nit: the comment is misleading as it's not aligned with the if condition...

Suggestion: "only send to broker if SERVICE_TEST is not "true"

Reply:
Sure.